demo-notebooks/testing_instructions.txt

```diff
@@ -0,0 +1,40 @@
+A tested version of Search & Download has been created, named "Main_Site_Search_&_Download-Python-new-testable-example.ipynb"
+Modifications were made to handle our example dataset and improve error messages, but the core features of the notebook remain unchanged. 
```

Comment:
I recommend adding the date that you copied "Search & Download - Python.ipynb" or the version of that notebook you copied.